city_scrapers/spiders/chi_infrastructure_trust.py

```diff
@@ -14,7 +14,7 @@ class ChiInfrastructureTrustSpider(CityScrapersSpider):
         "http://chicagoinfrastructure.org/public-records/meeting-records-2/",
         "http://chicagoinfrastructure.org/public-records/scheduled-meetings/"
     ]
-
+ 
```

Comment:
It looks like this is the issue, there's extra whitespace added to this line. Not sure why it isn't getting fixed by `yapf` locally, but if you remove it that should resolve the error

Reply:
@pjsier. Thank you. It manage to work now.